src/pages/index.js

```diff
@@ -1,6 +1,7 @@
 import Home from './home';
 import WatchList from './watchList';
 import SearchMovie from './searchMovie';
+import PlayerScreen from './playerScreen';
```

Comment:
Player screen can be deleted as we planned to open player in fullscreen mode only